tests/test_data.py

```diff
@@ -46,3 +46,39 @@ def test_looks_like_test_file_false(filename):
         from_imports={},
     )
     assert not state.looks_like_test_file()
+
+
+@pytest.mark.parametrize(
+    ("filename"),
+    (
+        "__init__.py",
+        "package/__init__.py",
+        "project/package/__init__.py",
```

Comment:
we should test windows style as well

```suggestion
        "__init__.py",
        "package/__init__.py",
        "package\__init__.py",
        "project/package/__init__.py",
        "project\package\__init__.py",
```